plugins/balloonpanel/plugin.js

```diff
@@ -755,7 +755,7 @@
 		 * @param {String} title A new panel title.
 		 */
 		setTitle: function( title ) {
-			this.parts.title.setHtml( title );
+			this.parts.title.setHtml( CKEDITOR.tools.htmlSafeByReview(title, 'Value controlled by developer, potential XSS risk if exposed to user. The title variable is passed as a parameter from the setTitle function which is part of the balloonpanel API accessible to the developer.') );
```

Comment:
Did we decide whether to do a legacy conversion here or not? `title` seems like it doesn't need to be HTML, but it very easily could be smuggling tags, so it seems a bit dangerous to mark it as safe by review. (It also could probably be safe to sanitize, assuming that no super-complex markup is likely making its way into here, which we can explore in addition to legacy conversions?)

Reply:
SGTM